.azure-pipelines/client.yml

```diff
@@ -85,8 +86,15 @@ jobs:
 
   - job: 'Test'
 
+    dependsOn:
+      - 'Publish'
+
     strategy:
       matrix:
+        Linux_Node6:
+          os.name: 'Linux'
```

Comment:
Can you please update the variable naming while you are changing these? Also I suspect it isn't worth having os.name as it adds little value vs the extra plumbing that we are doing for it.

Reply:
Donezo